frontend/src/components/Trigger.tsx

```diff
@@ -182,6 +183,11 @@ export default class Trigger extends React.Component<TriggerProps, TriggerState>
             value={maxConcurrentRuns}
             variant='outlined'
           />
+          {!this.props.isMaxConcurrentRunValid && (
+            <div className={classes(padding(20, 'r'))} style={{ color: 'red' }}>
+              {'Invalid input. The maximum concurrent runs number should be an integer.'}
```

Comment:
NIT: How do you think about the following error message?

`Invalid input: Maximum concurrent runs should be a positive integer.`